nano/node/transport/tcp.hpp

```diff
@@ -137,6 +137,10 @@ namespace transport
 			std::shared_ptr<nano::transport::channel_tcp> channel;
 			std::shared_ptr<nano::socket> socket;
 			std::shared_ptr<nano::bootstrap_server> response_server;
+			channel_tcp_wrapper (std::shared_ptr<nano::transport::channel_tcp> & channel_a, std::shared_ptr<nano::socket> & socket_a, std::shared_ptr<nano::bootstrap_server> & server_a) :
```

Comment:
minor: I think we pass by const ref when passing a shared_ptr by reference (same below)

Reply:
Good catch, did the same for the endpoint ref. Thanks!